quipucords/fingerprinter/task.py

```diff
@@ -335,34 +334,26 @@ def _process_details_report(self, manager_interrupt, details_report):
 
                     # Add auto-generated fields for the insights report
                     fingerprint_dict['id'] = fingerprint.id
-                    fingerprint_dict['system_platform_id'] = \
-                        fingerprint.system_platform_id
 
                     # Serialize the date
                     for field in SystemFingerprint.DATE_FIELDS:
                         if fingerprint_dict.get(field, None):
                             fingerprint_dict[field] = \
                                 date_field.to_representation(
                                     fingerprint_dict.get(field))
-                    fingerprint_dict['system_platform_id'] = \
-                        uuid_field.to_representation(
-                            fingerprint_dict.get('system_platform_id'))
                     final_fingerprint_list.append(fingerprint_dict)
 
                     # Check if fingerprint has canonical facts
                     for fact in CANONICAL_FACTS:
                         if fingerprint_dict.get(fact):
                             found_canonical_facts = True
                             break
-                    # If canonical facts, add it to the insights_hosts dict
-                    insights_id = fingerprint_dict.get(
-                        'system_platform_id')
                     if found_canonical_facts:
                         not_null_facts = {
                             'bios_uuid': 'bios_uuid',
                             'ip_addresses': 'ip_addresses',
                             'mac_addresses': 'mac_addresses',
-                            'insights_id': 'insights_client_id',
+                            'insights_client_id': 'insights_client_id',
```

Comment:
I think this should stay as `insights_id`

Reply:
Yep